src/common/obj_bencher.cc

```diff
@@ -17,6 +17,7 @@
  */
 #include <pthread.h>
 #include "common/Cond.h"
+#include "include/compat.h"
```

Comment:
@ukernel I'd suggest putting compat.h as very first include file.
This will prevent other include files from defining things that need to be repaired later.
Like ENODATA